src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -101,6 +101,13 @@ module sonic-vlan {
 						pattern "enabled|disabled";
 					}
 				}
+
+                leaf loopback_action {
+                    description "Packet action when a packet ingress and gets routed on the same IP interface";
+                    type string{
```

Comment:
Can we define a type in sonic-types and use it across the 4 files? Please check here for example

[    typedef boolean_type {](https://github.com/Azure/sonic-buildimage/blob/299e67a318ec65d1d4a060e8386477552cee29a9/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L241)

Reply:
Done.